cirq/api/google/v2/metrics.proto

```diff
@@ -0,0 +1,79 @@
+syntax = "proto3";
+
+package cirq.api.google.v1;
+
+option java_package = "com.google.cirq.api.google.v2";
+option java_outer_classname = "CalibrationProto";
+option java_multiple_files = true;
+
+// Performance metrics for a quantum processor.
+message Metrics {
+
+  // The time the metrics were collected, in milliseconds since Jan 1, 1970.
+  // Metric collection take time, so this is the time at which all of the
+  // metrics have been collected.
+  uint64 collection_time_ms = 1;
+
+  // Map from a metric type to a collection of the metrics all of this type.
+  // Keys should be snake_case and fairly descriptive of the metric.
```

Comment:
Why is the format encouraging a style guide on its keys?

Reply:
because standards for casing make you not have to remember whether it was T1_time or T1Time etc.

Reply:
Oh right this is coming from the server, so we can enforce things like that.